batch/batch/batch.py

```diff
@@ -1163,14 +1165,15 @@ def is_successful(self):
     params = request.query
     user = userdata['username']
     batches = await _get_batches_list(params, user)
-    token = new_csrf_token()
-    context = base_context(deploy_config, userdata, 'batch')
+    csrf_token = new_csrf_token()
+    session = await aiohttp_session.get_session(request)
+    context = base_context(deploy_config, session, userdata, 'batch')
     context['batch_list'] = batches[::-1]
-    context['token'] = token
+    context['csrf_token'] = csrf_token
```

Comment:
At least one page still appears to use "token" (workshop/admin.html)

<img width="830" alt="Screenshot 2019-09-25 18 22 59" src="https://user-images.githubusercontent.com/5543229/65644176-9996f480-dfc1-11e9-8c9a-55c2553bd280.png">